src/Compute/Compute/help/New-AzDiskEncryptionSetConfig.md

```diff
@@ -48,6 +48,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -EncryptionType
```

Comment:
It is very strange that argument completer is not generated here. Can you try to regenerate it here?